libunwind/include/libunwind.h

```diff
@@ -43,6 +43,12 @@
   #define LIBUNWIND_AVAIL
 #endif
 
+#if defined(__SANITIZE_MEMORY__) ||                                            \
```

Comment:
`__SANITIZE_MEMORY__` is a macro defined by the Linux kernel and copied to other projects.

We can drop it.

Reply:
Ah I assumed it was needed for GCC since it doesn't have __has_feature().

Reply:
GCC just doesn't support msan :)

Reply:
Sounds good. In that case I'll update the PR when I'm back in the office next Thursday.